packages/broker/src/plugins/operator/MaintainOperatorValueService.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger } from '@streamr/utils'
+import { MaintainOperatorValueHelper } from "./MaintainOperatorValueHelper"
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+const CHECK_VALUE_INTERVAL = 1000 * 60 * 60 * 24 // 1 day
+
+export class MaintainOperatorValueService {
+    private readonly config: OperatorServiceConfig
+    private checkValueInterval: NodeJS.Timeout | null = null
+    private readonly helper: MaintainOperatorValueHelper
+
+    constructor(config: OperatorServiceConfig) {
+        logger.trace('MaintainOperatorValueService created')
+        this.config = config
+        this.helper = new MaintainOperatorValueHelper(config)
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    start(penaltyLimitFraction?: bigint): void {
+        logger.info('MaintainOperatorValueService started')
+        this.checkValue(this.config.operatorContractAddress, penaltyLimitFraction)
+        this.checkValueInterval = setInterval(() => {
```

Comment:
Maybe we want to ensure that there can't be two `checkValue` calls to be running in paraller?

Could use `scheduleAtInterval` helper from @streams/utils to achieve this. By using it there would be `CHECK_VALUE_INTERVAL` milliseconds _between_ two requests, i.e. the waiting starts only after the previous task has been completed.

Reply:
Changed to scheduleAtInterval